internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,412 @@
+package acceptance
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccLakeviewDashboard(t *testing.T) {
+	workspaceLevel(t, step{
+		Template: `
+		resource "databricks_lakeview_dashboard" "d1" {
+			display_name			= 	"Monthly Traffic Report"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			file_path				=	"/Users/divyansh.vijayvergia/terraform-provider-databricks/Experimental-Divyansh/json_file"
```

Comment:
Don't use absolute paths, nor your username

Reply:
Yes. I forgot to change this here.